definitions/features/service.rb

```diff
@@ -10,37 +10,16 @@ def handle_services(spinner, action, options = {})
     if feature(:downstream) && feature(:downstream).less_than_version?('6.3')
       use_katello_service(action, options)
     else
-      options[:reverse] = action == 'stop'
-
-      filtered_services(options).each do |service|
-        spinner.update("#{action_noun(action)} #{service}")
-        perform_action_on_service(action, service)
-      end
-
-      spinner.update("All services #{action_past_tense(action)}")
+      use_system_service(action, options, spinner)
     end
   end
 
-  def list_services(service_list)
-    service_list = service_list.join(', ')
-    puts "#{service_list}\n"
-  end
-
-  def get_services_from_features(features)
-    features.map(&:services).
-      inject(&:merge).
-      sort_by { |_, value| value }.
-      map { |service| service[0] }
-  end
-
   def existing_services
-    service_list = get_services_from_features(available_features)
-    service_list.select { |service| service_remote?(service) || service_exists?(service) }
-  end
-
-  def service_remote?(service)
-    (service == 'postgresql' && !feature(:instance).postgresql_local?) ||
-      (service == 'mongod' && feature(:instance).database_remote?(:mongo))
+    ForemanMaintain.available_features.map(&:services).
+      flatten(1).
+      sort.
+      uniq(&:to_s).
```

Comment:
This leads to foreman's posgres and canclepin's service being treated as separate, even though especially with local, they are the same service:

```
- starting postgresql (foreman)Redirecting to /bin/systemctl start postgresql.service

| starting postgresql (candlepin)Redirecting to /bin/systemctl start postgresql.service
```

Also, there is missing space before `Redirecting…` string

Reply:
This was intentional. I see in this case it could be confusing. I'll try to change the logic to use the component only for non-local DBs which should fix the situation while not breaking the partly-remote scenario.

Reply:
yes, that would be good enough solution for me